server/views.py

```diff
@@ -288,21 +290,39 @@ def request(request, secret_id):
                                     reverse("server:approve", args=[new_request.id]),
                                 )
                                 if hasattr(settings, "EMAIL_SENDER"):
-                                	email_sender = (
-                                    	settings.EMAIL_SENDER
-                                	)
+                                    email_sender = (
+                                        settings.EMAIL_SENDER
+                                    )
                                 else:
                                     email_sender = (
                                         "requests@%s" % request.META["SERVER_NAME"]
                                     )
-                                send_mail(
-                                    "Crypt Key Request",
-                                    email_message,
-                                    email_sender,
-                                    [user.email],
-                                    fail_silently=True,
-                                )
-
+                                
+                                logger.error("[!] Sending request email to {} from {}".format(user.email, email_sender))
```

Comment:
I doubt these should be logged as an error

Reply:
Yeah, I did it to _very much definitely_ see the error. Agreed!